src/super_gradients/recipes/training_hyperparams/imagenet_mobilenetv2_train_params.yaml

```diff
@@ -6,7 +6,7 @@ max_epochs: 450
 lr_mode: step
 initial_lr: 0.032   # for total batch-size of 512
 lr_decay_factor: 0.973
-lr_updates:
+lr_updates: #FIXME: This creates a bug because a list is expected and it gets a np array
```

Comment:
hydra actually has a builtin function for that:
```lr_updates : ${range:100,200,50}```

Reply:
It's actually not builtin, it requires to add :
`OmegaConf.register_new_resolver("range", lambda *args: np.arange(*args).tolist())`

Reply:
ok. so keep it as it is